internal/fs/inode/dir_test.go

```diff
@@ -1210,3 +1229,30 @@ func (t *DirTest) CreateLocalChildFile_ShouldnotCreateObjectInGCS() {
 	AssertEq(nil, err)
 	AssertEq(nil, result)
 }
+
+func (t *DirTest) LocalFileEntriesEmpty() {
+	localFileInodes := map[Name]Inode{}
+
+	entries := t.in.LocalFileEntries(localFileInodes)
+
+	AssertEq(0, len(entries))
+}
+
+func (t *DirTest) LocalFileEntriesWith2LocalChildFiles() {
```

Comment:
Add unit test when all non child files are passed. this is to ensure that empty list is returned and not error or nil